Signal/src/ViewControllers/ContactsViewHelper.m

```diff
@@ -297,6 +297,35 @@ - (BOOL)doesContact:(Contact *)contact matchSearchTerms:(NSArray<NSString *> *)s
 
 #pragma mark - Editing
 
+- (UIAlertController *)missingContactAccessAlertController
```

Comment:
Why not `showMissingContactAccessAlertController:(UIViewController *)fromViewController` and present this vc as well?